pkg/gce-pd-csi-driver/controller.go

```diff
@@ -152,7 +152,7 @@ func (gceCS *GCEControllerServer) CreateVolume(ctx context.Context, req *csi.Cre
 	content := req.GetVolumeContentSource()
 	if content != nil {
 		if content.GetSnapshot() != nil {
-			snapshotId = content.GetSnapshot().GetId()
+			snapshotId = content.GetSnapshot().GetSnapshotId()
```

Comment:
Add a TODO or open a bug to add support for "Volume Source" (cloning) which is new in 1.0.